Esri/arcgis-maps-sdk-kotlin-samples
# show viewshed location

## Description

PR to add a new Kotlin sample _"Show Viewshed from point in scene"_ in `Analysis` category.

## Links and Data

Sample Epic: `runtime/kotlin/issues/1137`

## Note
The double touch drag feature is not available in kotlin api as of 200.2.0.3921. Due to which the `move viewshed` functionality isn't available with this sample


## What To Review

-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files


## How to Test

Run the sample for build 200.2.0:3910 or below on the sample viewer or the repo.
There is an issue with viewshed not showing for builds 3911 and above. See [comment here](https://esri-runtime.slack.com/archives/CBT4CTZ0X/p1687902640429399)
